sdk/js/src/types.ts

```diff
@@ -47,6 +47,7 @@ export interface DVCOptions {
     enableEdgeDB?: boolean
     logger?: DVCLogger
     logLevel?: DVCDefaultLogLevel
+    workerProxyURL?: string
```

Comment:
should this option be named something more user-facing? The fact that we're using "workers" is sort of an implementation detail. Maybe `cloudProxyURL` or `apiProxyURL`?

Reply:
yea `apiProxyURL` could make sense.